lib/ansible/modules/monitoring/zabbix_host.py

```diff
@@ -239,6 +247,9 @@ def update_host(self, host_name, group_ids, status, host_id, interfaces, exist_i
                 parameters['proxy_hostid'] = proxy_id
             if visible_name:
                 parameters['name'] = visible_name
+            if description:
```

Comment:
as a response to your question: change this condition to check for not None. So an empty str will be handeled.

```
if description is not None:
```